assembly-multiuser/assembly-wsmaster-war/src/main/java/org/eclipse/che/api/deploy/MultiUserCheWsMasterModule.java

```diff
@@ -87,6 +91,6 @@ protected void configure() {
                 + "-addr :4412 "
                 + "-cmd ${SHELL_INTERPRETER} "
                 + "-enable-auth "
-                + "-logs-dir $HOME/che/exec-agent/logs");
+                + "-logs-dir $(eval \"$" + LOGS_DIR_SETTER_VARIABLE + "\"; echo \"$" + LOGS_DIR_VARIABLE + "\")");
```

Comment:
Can you elaborate on why we need this change?

Reply:
see the comment of commit 4bda7b4c9a709483eff25cc0a7628ffaa6cd0c3a for a more detailed explanation